Analysis/DEEP2_R23_O32/Plotting/te_metal_plots.py

```diff
@@ -18,20 +18,56 @@
 from Metallicity_Stack_Commons.column_names import filename_dict
 
 jiang_coeff = [-24.135, 6.1532, -0.37866, -0.147, -7.071]
+bian_coeff = [138.0430, -54.8284, 7.2954, -0.32293]
 
-a = -24.135
-b = 6.1532
-c = -0.37866
-d = -0.147
-e = -7.071
 
+def jiang_calibration(metal_det, lo32):
+    """
+    Purpose
+    Calculating the Jiang calibration for the metallicity and O32 measurements enter
+
+    Parameters
+    metal_det -> dictionary of 12+log(O/H)
+    lO32 -> log(O32) value
+    """
+    return jiang_coeff[0] + jiang_coeff[1] * metal_det + jiang_coeff[2] * (metal_det * metal_det)\
+           + jiang_coeff[3] * (jiang_coeff[4] + metal_det) * lo32
+
+
+def bian_calibration_r23(metal_det):
+    """
+    Purpose
+    Calculating the R23 of Bian calibration for the metallicity measurements enter
```

Comment:
Provide the full reference for Bian et al.

Reply:
fixed